server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -115,13 +117,23 @@ object Server extends AkkaBackCompat {
   Slf4jSetup.init()
 
   val log = LoggerFactory.getLogger("Server")
+  val XDG_CONFIG_HOME = " ~/.config/ensime-server"
+
+  lazy val shutDownOnDisconnect = Option(System.getProperty("ensime.explode.on.disconnect")).isDefined
```

Comment:
if we're using typesafe config you can move all of these into a `reference.conf`